src/components/MainNavbar/index.js

```diff
@@ -1,4 +1,5 @@
 import React, {Component } from 'react';
+import {BrowserRouter as Router, Route, Link, Redirect} from 'react-router-dom';
```

Comment:
Why do you import all things form 'react-router-dom'? React throws warnings 'unused import'.